frontend/fragments/try-footer.html

```diff
@@ -90,7 +90,7 @@ <h2 id="styles"> Styles </h2>
   <td><code>https://img.shields.io/badge/style-flat--squared-green.svg?style=flat-square</code></td>
   </tr>
   <tr>
-  <td><img src="/badge/style-for--the--badge-green.svg?style=for-the-badge" alt=''/></td>
+  <td><img src="/badge/style-for--the--badge-green.svg?style=for-the-badge" alt='' style='height:28px;'/></td>
```

Comment:
Huh. Why is this needed? Are the images fixed height? Would `height:auto` work?

Reply:
all the images in tables are `height:20px`
![image](https://user-images.githubusercontent.com/7288322/32303989-fa118568-bfd0-11e7-9116-a359a694d8af.png)
I could change this badge to `height:auto` if you like though.

From memory the change to `height:20px` was due to the page used to jump around a lot while loading.

Reply:
Could update the css in `try-header` to remove `.badge-img img` which would only affect the following sections:
![image](https://user-images.githubusercontent.com/7288322/32304330-51824f6a-bfd3-11e7-8423-a9c14fd17974.png)
![image](https://user-images.githubusercontent.com/7288322/32304334-5a8c7234-bfd3-11e7-9166-18b8d938b370.png)
```css
table.badge > tbody > tr > td > img, #copyImg {
  height: 20px;
  vertical-align: middle;
}
```

Reply:
Nah, thanks for explaining. I remember that change and it makes sense now.